### ISSUE TITLE
code

### ISSUE THREAD

**Issue by Djerry-h** [jhkim611/CLAD#3]
Hello, I looked at the code and I'm curious, it seems that you didn't use a test set for the final prediction. I'm not sure if I missed it or if there's a reason for this. Could you help me point out the reason?"
Would you like me to address this concern directly, or do you need further assistance with a specific code snippet or context?

**Comment by jhkim611** [comment]
Sorry for getting back so late. If you're referring to the GCN, the point of the model is to obtain softmax probabilities (i.e., logits) to use for the JSD+ metric. Thus, we only need a training set to train the GCN, as we can use the logits put out for every node after training.
